src/ml-service/packages/sensor_location/app/templates/index.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Index</title>
+  <meta name="description" content="The HTML5 Herald">
```

Comment:
replace the meta tags content values such as The HTML5 herald,  author, sitepoint to actual values, you can remove th

Reply:
Just seen this, we might actually not need these template files since this is a backend part of our system. Could we just delete them? :)